packages/cockroach/extra/register.py

```diff
@@ -278,21 +278,69 @@ def _init_cockroachdb_cluster(
     """
     @retrying.retry(
         wait_fixed=1000,
-        stop_max_attempt_number=10,
+        stop_max_attempt_number=60,
```

Comment:
Why did we change this to 60?

Reply:
My thinking...: when this program crashes we hand back control to systemd. This change delays this. Why do we want this? It probably makes sense, but I'd love to read about a reason :-)

Reply:
If the initialization takes longer than 10 seconds DC/OS would never be able to start up.